contracts/exchange/interfaces/IRoyaltyManager.sol

```diff
@@ -34,7 +39,15 @@ interface IRoyaltyManager {
         uint256[] calldata _royaltyFees
     ) external;
 
+    function transferRoyalty(
+        uint256 _orderId,
+        address _receiver,
+        uint256 _fee
+    ) external;
+
     function transferPlatformFee(address _sender, address _token, uint256 _total) external;
 
     function transferPlatformFee(address _token, uint256[] calldata _orderIds, uint256[] memory platformFees) external;
+
+    function transferPlatformFee(address _token, uint256 _orderId, uint256 _total) external;
```

Comment:
make the single one "transferPlatformFee" and the batch function "transferPlatformFees"